auth/index.html

```diff
@@ -1163,7 +1164,12 @@ <h2>Message log</h2>
 
             TKHQ.setParentFrameMessageChannelPort(iframeMessagePort);
 
-            await TKHQ.initEmbeddedKey();
+            if (event.data.dangerouslyOverrideIframeKeyTtl) {
+              await TKHQ.initEmbeddedKey(event.data.dangerouslyOverrideIframeKeyTtl);
+            } else {
+                await TKHQ.initEmbeddedKey();
+            }
```

Comment:
Let's do validation to make sure the TTL passed is a number. In JS a lot of stuff is truthy.

I'd also like to simplify TKHQ.initEmbeddedKey so that it always expects an argument (the TTL). And here we can have:
```js
var iframeKeyTtl = TURNKEY_EMBEDDED_KEY_TTL_IN_MILLIS
if (event.data.dangerouslyOverrideIframeKeyTtl && ...validation...) {
    iframeKeyTtl = event.data.dangerouslyOverrideIframeKeyTtl;
}

await TKHQ.initEmbeddedKey(iframeKeyTtl);
```